.github/workflows/build.yml

```diff
@@ -493,6 +506,31 @@ jobs:
       - name: Compute test coverage
         uses: codecov/codecov-action@v3
 
+  job_profiling_node_unit_tests:
```

Comment:
I followed the convention for bun and deno, but lmk if it doesn't make sense and you'd rather have this inside the test job (we just need to ensure bindings are build before running tests)

Reply:
Can we only run unit tests/build and test for profiling node if we make changes to profiling node file? We prob need to do this for deno and bun as well fwiw.

https://github.com/getsentry/sentry-javascript/blob/28b2eb1755ff93b54f5b396b22748e7a7acbc7b6/.github/workflows/build.yml#L147-L154